libraries/radwordsprocessing/concepts/fields/fields.md

```diff
@@ -59,6 +59,8 @@ In the document object model Fields are represented by the [Field](https://docs.
 
 * **[Section Pages]({%slug radwordsprocessing-concepts-sectionpages-field%})**
 
+* **[Sequence Field]({%slug radwordsprocessing-concepts-sequence-field%})**
```

Comment:
It is a SeqField, not SequenceField. It is written both ways in MS documentation, but we (the team) decided to name it SeqField in our API.